app-header/app-header.html

```diff
@@ -280,8 +288,8 @@
        *
        * @type {number}
        */
-      _deltaHeight: 0,
-      
+      _ΔHeight: 0,
```

Comment:
Are we using special characters in code now? Given that they're hard to type, should probably just be `dHeight` or `deltaHeight`.